File: src/main/java/com/partymakers/shareparty/data/persistence/friends/FriendPersistenceRepository.java
package com.partymakers.shareparty.data.persistence.friends;

import com.partymakers.shareparty.data.persistence.friends.entity.FriendEntity;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface FriendPersistenceRepository extends JpaRepository<FriendEntity, String> {
}

File: src/main/kotlin/com/partymakers/shareparty/data/persistence/friends/FriendPersistenceRepository.kt
package com.partymakers.shareparty.data.persistence.friends

import com.partymakers.shareparty.data.persistence.friends.entity.FriendEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface FriendPersistenceRepository : JpaRepository<FriendEntity, String> {

    fun findByNickName(nickName: String): FriendEntity?
}